accounting/fiscal_localizations/localizations/chile.rst

```diff
@@ -0,0 +1,627 @@
+instructions on each section.
+
+.. image:: media/Chile02.png
+   :align: center
+   
+Electronic Invoice Data
+~~~~~~~~~~~~~~~~~~~~~~~
+This is part of the main information required to generate electronic Invoice, select your
+environment and the legal information, as well as the email address to receive invoices
+from your vendors and the alias you use to send invoices to your customers.
+
+.. image:: media/Chile03.png
+   :align: center
+
+Configure DTE Incoming email server
+***********************************
+In order to receive you the claim and acceptance emails from your customers it is
+crucial to define the DTE incoming email server, considering this configuration: 
+
+.. image:: media/Chile03_2.png
+   :align: center
+
+.. tip::
+   For your Go-live make sure you archive/remove from your inbox all the emails
+   related to vendor bills that are not required to be processed in Odoo. 
+   
+Certificate
+~~~~~~~~~~~
+In order to generate the electronic invoice signature, a digital certificate with extension(.pfx)
+is required, proceed to this section and load your file and password. 
+
+.. image:: media/Chile03_3.png
+   :align: center
+
+.. image:: media/Chile04.png
+   :align: center
+
+A certificate can be shared between several users, if this is the case leave the user field empty,
+so all your billing users can use the same one. On the other hand,  if you need to restrict the
+certificate for a particular user, just define the users in the certificate. 
+
+Financial Reports
+~~~~~~~~~~~~~~~~~
+The report Propuesta F29 requires two values that need to be defined as part of the company
+configuration: 
+
+.. image:: media/Chile05.png
+   :align: center
+
+Multicurrency
+~~~~~~~~~~~~~
+The official currency rate in Chile is provided by the SBIF, you can find the SBIF
+synchronization service in the currency rate configuration and you can set a predefined interval
+for the rate updates.  
+
+.. image:: media/Chile05_2.png
+   :align: center
+   :scale: 25
+
+In order to configure your token, you need to enable developer mode and access to
+:menuselection:`Settings --> Technical --> System parameters` and search the parameter
+``sbif_api_key`` and in the field Value introduce your token provided by the SBIF:
+
+.. image:: media/Chile05_3.png
+   :align: center
+   
+Chart of accounts
+~~~~~~~~~~~~~~~~~
+The chart of accounts is installed by default as part of the set of data included in
+the localization module, the accounts are mapped automatically in: 
+
+- Taxes.
+- Default Account Payable.
+- Default Account Receivable.
+- Transfer Accounts.
+- Conversion Rate.
+
+
+Master Data
+-----------
+
+Partner
+~~~~~~~
+Identification Type and VAT
+***************************
+As part of the Chilean localization, the identification types defined by the SII
+are now available on the Partner form, this information is essential for most transactions.
+
+.. image:: media/Chile06.png
+   :align: center
+   :scale: 25
```

Comment:
Is the scale a way to make the image tinier?
Try to change the height instead. It should work

Reply:
Image updated in the last commit.